sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1094,6 +1142,29 @@ std::vector<node> get_root_nodes() const;
 ----
 |Returns a list of all nodes in the graph which have no dependencies.
 
+|
+[source,c++]
+----
+event get_event(const node& node);
+----
+|Returns a SYCL event which represents the completion of node `node` which is
+valid only for the next execution of the graph. This event can be used as a
+dependency in the same way as normal SYCL events.
```

Comment:
Wondering if we should have the same wording as `update()` here around blocking behaviour

> The implementation may perform a blocking wait during this call on any in-flight executions of that same graph if required by the backend.

Since until we've done more implementation work I don't think we'll be able to say for sure whether this can be non-blocking.

Reply:
Following internal discussion:

- `get_event()` would be forced to sync if updating the event in the backend structures is required to provide a unique event per execution.
- The following example illustrates a more general problem with the event returned from `get_event()` being unique per execution:
-  ```c++
   sycl::event EventA = ExecGraph.get_event(SomeNode);
				
   Queue.ext_oneapi_graph(ExecGraph);
   sycl::event EventB = ExecGraph.get_event(SomeNode);
   // We haven't waited on eventA so it is reset and now valid only for the second execution
   // and is the same as eventB?
   Queue.ext_oneapi_graph(ExecGraph);
				
   EventA.wait();
   EventB.wait();
```
- Implementing this so that events are unique would require a host-sync when `EventB` is obtained to update the backend structures with a new event. This would render `EventA` complete anyway.
- In either case effectively this means that events are only valid for the most recent execution of the graph, so we should make this the official wording. If a user requires to wait specifically on each event triggering they must enforce the ordering somehow, or use separate executable graphs to avoid event reset.